index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <form>
+        <div>
+          <label>Enter something</label>
+          <input autofocus />
+        </div>
+        <button type="submit">Make docx</button>
+    </form>
+    <script>
```

Comment:
Is there any reason why you keep this script code here? 
Why not in separated file?

Reply:
This is only first try whether `docx` library works well. All scripts and forms will be removed shortly.